frontend/tests/acceptance/pages/loginPage.js

```diff
@@ -3,13 +3,33 @@ const { I } = inject();
 module.exports = {
   url: '/login',
   fields: {
-    email: '//label[contains(text(),"email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    password: '//label[contains(text(),"password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    email: '//label[contains(text(),"Email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
```

Comment:
```suggestion
    email: '//label[contains(text(),"Email")]/parent:://input[contains(@class, "MuiInputBase-input")]',
```
similar for password

Reply:
the xpath willnot be valid
```//label[contains(text(),"Email")]/parent::div//input[contains(@class, "MuiInputBase-input")]``` can be used